generate_site.sh

```diff
@@ -5,7 +5,11 @@ function die()
     exit 1
 }
 
-[[ $# -ne 2 ]] && die "Not enough arguments"
+if (( $# != 2 ))
+then
+  die "Not enough arguments"
```

Comment:
Could we add an explicit help message like:

> Please enter a number of posts and a destination folder.

Reply:
Done.